### PR TITLE
several fixes for autologin - new selenium and xpath

### PR DESCRIPTION
**Pull request by ReuschelCGN**

### CODE REVIEW

**Comment by acocalypso** [comment]
I just checked out this pr and got the following issue:

```
aco@v220220315612181906:~/intelwatcher$ ../intelwatcher_venv/bin/python3 intel_watcher.py
Traceback (most recent call last):
  File "intel_watcher.py", line 15, in <module>
    from intelwatcher.get_cookie import mechanize_cookie, selenium_cookie
  File "/home/aco/intelwatcher/intelwatcher/get_cookie.py", line 212
    driver.find_element("xpath", "//div[@id="dashboard_container"]//a[@class="button_link" and contains(text(), "Facebook")]").click()
                                             ^
SyntaxError: invalid syntax
```

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    restart: always
+    volumes:
+      - /etc/localtime:/etc/localtime:ro
+      - ./intelwatcher/config.ini:/usr/src/app/config.ini
+      - ./intelwatcher/cookie.txt:/usr/src/app/cookie.txt
+    labels:
+      ofelia.enabled: "true"
+      ofelia.job-exec.intelwatcher.schedule: "@every 48h"
+      ofelia.job-exec.intelwatcher.command: "python3 intel_watcher.py"
+    depends_on:
```
Comment (Fabio1988) [inline docker-compose.yml]: pls remove that depends_on, not everyone uses rocketdb :)

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+
+  intelwatcher:
+    container_name: intelwatcher
+    build: ./intelwatcher
+    tty: true
+    restart: always
+    volumes:
+      - /etc/localtime:/etc/localtime:ro
+      - ./intelwatcher/config.ini:/usr/src/app/config.ini
```
Comment (Fabio1988) [inline docker-compose.yml]: pls remove relative path here too, same with cookie

`- ./config.ini:/usr/src/app/config.ini`

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    image: mcuadros/ofelia:latest
+    restart: always
+    command: daemon --docker
+    volumes:
+      - /etc/localtime:/etc/localtime:ro
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+
+  intelwatcher:
+    container_name: intelwatcher
+    build: ./intelwatcher
```
Comment (Fabio1988) [inline docker-compose.yml]: `build: .`

File: docker-compose.yml (hunk 1 above)
Comment (ReuschelCGN) [reply docker-compose.yml]: removed

File: docker-compose.yml (hunk 2 above)
Comment (ReuschelCGN) [reply docker-compose.yml]: fixed

File: docker-compose.yml (hunk 3 above)
Comment (ReuschelCGN) [reply docker-compose.yml]: also fixed

**Comment by ccev** [comment]
I think you would be better off removing Dockerfile and docker-compose.yml alltogether. That's exceeds the scope of this PR.

**Comment by acocalypso** [comment]
Login with Facebook seems to work.

While scraping this issue occurs:
```
'IntelMap' object has no attribute 'r'
Traceback (most recent call last):
  File "/home/aco/intelwatcher/intelwatcher/ingress.py", line 136, in
scrape_tiles
    result = self.r.post("https://intel.ingress.com/r/getEntities",
json=data, headers=self.headers,
AttributeError: 'IntelMap' object has no attribute 'r'```

**Comment by Fabio1988** [comment]
> Login with Facebook seems to work.
> 
> While scraping this issue occurs:
> 
> ```
> 'IntelMap' object has no attribute 'r'
> Traceback (most recent call last):
>   File "/home/aco/intelwatcher/intelwatcher/ingress.py", line 136, in
> scrape_tiles
>     result = self.r.post("https://intel.ingress.com/r/getEntities",
> json=data, headers=self.headers,
> AttributeError: 'IntelMap' object has no attribute 'r'```
> ```

Exact same error, with or without docker execution.
https://github.com/SenorKarlos/intelwatcher/blob/b54931d8176775fe2c24e35d008d75e347bc4654/intelwatcher/ingress.py#L136

**Comment by acocalypso** [comment]
cookie issue? My cookie looks different ad when I get it manually.

**Comment by acocalypso** [comment]
Google Login with selenium and Chrome (no docker) is working fine for me

**Pull request closed without merging**